spec/models/invoice_spec.rb

```diff
@@ -6,7 +6,7 @@
     it { should have_many :invoice_items }
     it { should have_many :transactions }
     it { should have_many(:items).through(:invoice_items) }
-    # it { should have_one(:merchant).through(:items) }
```

Comment:
Did we remove the has_one relationship in the model or is this just tested differently?